### ISSUE TITLE
Fixes / Improvements WEB

### ISSUE THREAD

**Issue by KostyaEver** [ever-co/ever-teams#341]
- [x] For all combo boxes Tabs Open & Closed - add task counters (not work now)
- [x] The main Timer does NOT reset time to 00:00 when the user changes to another team or selects another task
- [x] The main Timer - it is not tracking time when the user got to another pagan web Browser - The user should be able to track time when the user is surfing other pages or other software
- [x] Add a Tooltip to the main Timer when it is in non-active mode and user is trying to click on it - "Please, select or create a new task to start tracking the time"
- [x] In the main Timer - let's replace the icon 'pause' to icon "stop" 
- [x] Estimate on user card in edit mode, let's add inside inputs for hours input - 'h' and for minutes input - 'm' (not like now empty)
- [x] Edit task title on user card mode - let's change the size and font of the task text - make it bigger AND once you edited it to save changes user can click out of card and it saves changes (now user presses 'Enter' and it saves)
- [x] Header fix logo position - make it more centralised
- [x] Rename column 'Total worked 24 hours' to 'Total worked Today' 
- [x] Estimate under the main input - when user enters minutes - delete all data in the input so that user add exactly that he/she wants (sure we should add '0' if the user estimate from 1. to 9 minutes)
- [x] Estimate block under the main input / save data - as an additional option let's make if the user adds values and clicks out of the card it saves estimation
- [x] Estimate in (...) menu should switch on 'edit' mode the same as 'pen' button in Estimation column
- [x] Progress Bar - fix if the user continues tracking time overestimated time (progress bar should be fixed size) and the same task should display the. same progress Bar for Task page
- [x] User card 'Worked on task' column - add two values 'Today X h X m | 'Total X h X m'
- [x] Avatar Popup menu - rename 'Profile' to 'Tasks'
- [x] Once a user creates Team first time on WEB page 'Create team page 2' - the cursor should be automatically set in the Name input
- [x] Task page on current Task card - progress bar displays progress only after user stopped the timer
- [x] Estimation on user card - edit mode - replace ‘/’ by ‘:’
- [x] User card - once the user changes the estimate and clicks out of the card - it does not save a new estimate
- [x] User card - once user changes estimate under main Input and saves it by pressing ‘Enter’ - should remove the cursor from the inputs ‘Hours or Minutes’ when user pressed Enter
- [x] User card ( … ) 'Edit' button does not work - should set 'Task title' on user card to edit mode
- [x] User card ( … ) Assign task button only  opens the Combobox menu, we should be able to create task the same as in the main input
- [x] User see Open list tasks after closed a task - and should see empty main input with uncollapsed Combobox (list of open tasks) and nothing on own card. Related to #344 
- [x] Invite user API integration
- [ ] #366
- [ ] Invite card - let's add some fake grey color data: in column 'Status' - grey circle,  column 'Name' - button 'Invite' (as it is now) in column 'Task' - 'Task Title', column 'Worked on Task' - 00:00, column 'Estimate' - Grey line (as it is now), column 'Total worked Today' - 00:00.
- [ ] Task page - add per each card button ‘start’ inside ‘Estimate’ column next to ‘Progress Bar’
- [ ] Task page - add per each card ( ... ) menu
- [x] Team page - fix ( ... ) menu (not work)
- [ ] Live time update of 'Time' for ‘Total worked Today’ and ‘Worked on Task’ on user card (not when user stops timer)
- [x] When a user is typing in the main input - tabs ‘open’ & ‘closed’ should display correct data
- [ ] Once user closed task - the cursor should focus in the main input  (in order to type something)
- [x] Once user selected task - combobox should be collapced (in order to estimate task or something else)
- [ ] Once user estimates a task first time - when sets the cursor in estimate h/m inputs - should erase all data inside. When the user edit current estimate and presses ‘Estimate’ ( … ) or ‘pen’  on user card AND - the cursor focus should be right after a digit (at the end of input) and digit should be in highlighted mode in order to easy delete previous data.
- [ ] Task page - user should be able to filter tasks by status
- [ ] Task / Team pages - user pressed 'Start' tracking time for a task, status of the task should become 'in progress'
- [ ] Team page invited user view - add per each card ( ... ) menu
- [ ] Team page invited user view - does not display in Tab number of tasks open
- [ ] Team page invited user view - does not display all tasks
- [ ] Team page invited user view - if user selects the same task, it displays the wrong estimate 
- [ ] Team page invited user view - the user is not able to track the time
- [ ] Invite page - once the user 'resend' the code - should make empty code cells
- [ ] Invite page - after completing password things - user should see Gauzy Teams main page
- [ ] Invite page - complete registration - message that password is wrong should be later when both inputs are filled in wrong way
- [x] #294 
- [ ]  Team page - once the user confirmed the estimation under the main input (clicking out of estimate input) it should not update the estimate more times until the user sets the cursor to the estimated input again
